app.ts

```diff
@@ -1,19 +1,29 @@
 import express from "express";
 import dotenv from "dotenv";
 import morgan from "morgan";
-import { router as routes } from "./controllers/server"
+import mongoose from "mongoose";
+import {router as healthRoutes} from "./controllers/health";
```

Comment:
right now it does... but the app.ts file refers to it as the file which holds the routes and doesn't know how many routes exist - this name is more general. And this will be good in the future when other ___Routes files will exist